### PR TITLE
Lightflow

### PR DESCRIPTION
**Pull request by jrmlhermitte**
Add lightflow (should add to latest collection env 2018-2.0 during deployment)

### CODE REVIEW

**Comment by jrmlhermitte** [comment]
oops, commits included other unrelated files. i deleted them and restarted. this is good to go. thanks!

File: recipes-tag/lightflow/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+build:
+  number: 0
+  script:
+    - python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+        - numpy x.x
```
Comment (mrakitin) [inline recipes-tag/lightflow/meta.yaml]: Do we need `x.x` here at all? Same for line 23.

Comment (jrmlhermitte) [reply recipes-tag/lightflow/meta.yaml]: done. thanks!

**Comment by jrmlhermitte** [comment]
Andreas has updated lightflow to a new version. it fixes a memory bug and allows for networkx 2.1!!
Ready to merge!

**Comment by tacaswell** [comment]
You only need to worry about pinning numpy if you complied c-extensions, if you are only using numpy from the python level there are no problems!

**Comment by tacaswell** [comment]
I merged this and then noticed it does not depend on networkx...

**Comment by jrmlhermitte** [comment]
darn, i uploaded the branch again with an update, but I'm unable to build lightflow now:
```
conda_build.exceptions.DependencyNeedsBuildingError: Unsatisfiable dependencies for platform linux-64: {"colorlog[version='>=2.10.0']", "celery[version='>=4.0.2']", "pymongo[version='>=3.6.0']", "ruamel.yaml[version='>=0.14.2']"}
```
I need to look at this later this afternoon...

**Pull request merged**